src/main/java/org/powerflows/dmn/io/yaml/YamlDecsionConverter.java

```diff
@@ -17,13 +17,92 @@
 package org.powerflows.dmn.io.yaml;
 
 import org.powerflows.dmn.engine.model.decision.Decision;
+import org.powerflows.dmn.engine.model.decision.field.Input;
+import org.powerflows.dmn.engine.model.decision.field.Output;
+import org.powerflows.dmn.engine.model.decision.rule.Rule;
 import org.powerflows.dmn.io.DecisionToExternalModelConverter;
 import org.powerflows.dmn.io.yaml.model.YamlDecision;
+import org.powerflows.dmn.io.yaml.model.field.YamlFields;
+import org.powerflows.dmn.io.yaml.model.field.YamlInput;
+import org.powerflows.dmn.io.yaml.model.field.YamlOutput;
+import org.powerflows.dmn.io.yaml.model.rule.YamlRule;
+import org.powerflows.dmn.io.yaml.model.rule.YamlRuleEntry;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public class YamlDecsionConverter implements DecisionToExternalModelConverter<YamlDecision> {
     @Override
     public YamlDecision to(final Decision decision) {
-        throw new UnsupportedOperationException("This is not supported yet");
+        final YamlDecision yamlDecision = new YamlDecision();
+        yamlDecision.setId(decision.getId());
+        yamlDecision.setName(decision.getName());
+        yamlDecision.setExpressionType(decision.getExpressionType());
+        yamlDecision.setHitPolicy(decision.getHitPolicy());
+        yamlDecision.setFields(createFields(decision.getInputs(), decision.getOutputs()));
+        yamlDecision.setRules(decision.getRules()
+                .stream()
+                .map(this::ruleToYamlRule)
+                .collect(Collectors.toList()));
+        return yamlDecision;
```

Comment:
please meke empty line before.

Reply:
In the future, would be good to do simple builders for that, even if is not for end user.

Reply:
line added.
As for builders... this increases number of classes to load and keep in memory.
We can consider that but here I don't find it necessary